lxd/db/instances.go

```diff
@@ -271,7 +271,7 @@ SELECT nodes.id, nodes.address
 // string, to distinguish it from remote nodes.
 //
 // Containers whose node is down are addeded to the special address "0.0.0.0".
-func (c *ClusterTx) GetInstanceNamesByNodeAddress(project string, filter InstanceFilter) (map[string][]string, error) {
+func (c *ClusterTx) GetInstanceNamesByNodeAddress(projects []string, filter InstanceFilter) (map[string][]string, error) {
```

Comment:
I think @masnax and @tomponline may have some opinions on the best approach for this commit but I don't know if the other changes needed in this PR will impact this.